tests/integration/helpers.py

```diff
@@ -151,12 +151,44 @@ def project_exists(self, project_name):
         if location is _sentinel:
             location = self.example_repo
         repository_id = self.uuid4()
-        repository_iri = self.resource("repositories", repository_id)
+        repository_iri = self.resource("stacks", repository_id)
         await self.insert_node(Node(repository_iri, {
-            RDF.type: Doap.GitRepository,
+            RDF.type: Doap.Stack,
             Mu.uuid: repository_id,
             Doap.location: location,
         }))
+
+        drc_id = self.uuid4()
+        drc_text = """
+            version: \"2\"
+            services:
+
+              service1:
+                image: busybox
+                command: \"sleep 60\"
+
+              service2:
+                image: busybox
+                command: \"sleep 60\"
+        """
+        d_iri = IRI("http://stack-builder.big-data-europe.eu/resources/"
+        drc_iri = d_iri + "%s/%s" % ("docker-composes", drc_id)
```

Comment:
Python is not optimized for string concatenation. It's usually considered good practice to avoid it when possible:
```python
drc_iri = "%sdocker-composes/%s" % (d_iri, drc_id)
# or:
drc_iri = "".join([d_iri, "docker-composes/", drc_id])
```